src/code/entities/bank3.asm

```diff
@@ -9335,38 +9372,43 @@ GetEntityYDistanceAwayFromLink::
     ld   d, a                                     ; $7EFC: $57
     ret                                           ; $7EFD: $C9
 
-func_003_7EFE::
+; Inputs:
+;   bc   entity index
+;
+; Outputs:
+;   e   entity's direction to Link
```

Comment:
0 = right 1 = left 2 = up, 3 is down?
Might be nice to document that, or if there are some consts that are used for this.

Reply:
There are the DIRECTION_* constants. I've edited the comment to mention them.